controllers/addon_controller.go

```diff
@@ -155,7 +146,6 @@ func (r *AddonReconciler) execAddon(ctx context.Context, req reconcile.Request,
 		err := r.Finalize(ctx, instance, wfl, finalizerName)
 		if err != nil {
 			reason := fmt.Sprintf("Addon %s/%s could not be finalized. %v", instance.Namespace, instance.Name, err)
-			r.recorder.Event(instance, "Warning", "Failed", reason)
```

Comment:
Why are we removing the Events for addon?

Reply:
After moving to Resource (workflow) Add/Update trigger addon status update, this event recorder is not accurate.

Reply:
Addon status Events are part of the current expected behavior and a requirement, anytime the Addon status is updated the change message should be mirrored as an Event; let's restore that behavior.

Reply:
Yes, add it back and fixed accordingly 
https://github.com/keikoproj/addon-manager/pull/99/files#diff-6d74ffc9c209b277865197e4320441852fdfa7a53f06b570eea7bb3070f6735dR42